flocker/node/agents/blockdevice.py

```diff
@@ -1647,7 +1649,8 @@ def is_existing_block_device(dataset_id, path):
                     pass
 
         if ICloudAPI.providedBy(self._underlying_blockdevice_api):
-            live_instances = self._underlying_blockdevice_api.list_live_nodes()
+            live_instances = list(
+                self._underlying_blockdevice_api.list_live_nodes())
```

Comment:
:grey_question: I personally prefer `dict(a=1, b=2).keys()` over `list(dict(a=1, b=2))` everything else being equal. Seems it's only 1 more character, but seems to more explicitly express intent.

TBH, I always think that turning a dictionary into a list should have the behavior of `dict().iteritems()`, and the mental burden of remembering that I'm wrong is tiresome :stuck_out_tongue:. Optional as this is obviously personal preference.